Cognite.Simulator.Tests/IntegrationTests/ConnectorRuntimeTest.cs

```diff
@@ -137,8 +141,24 @@ public async Task TestConnectorRuntime()
                             }
                         }
                     ).ConfigureAwait(false);
+
+                    var simulator = await TestCdfClient.Alpha.Simulators.ListAsync(
+                        new SimulatorQuery
+                        {
+                           
+                        }
+                    ).ConfigureAwait(false);
+
+                    var unitQuantities = SeedData.SimulatorCreate.UnitQuantities;
                     var existing = integrations.Items.FirstOrDefault(i => i.ExternalId == ConnectorExternalId);
-                    Assert.True(existing.ExternalId == ConnectorExternalId);
+                    Assert.True( existing.ExternalId == ConnectorExternalId);
+                    
+                    var simulatorDefinition = simulator.Items.FirstOrDefault(i => i.ExternalId == SeedData.TestSimulatorExternalId);
+                    Assert.NotNull(simulatorDefinition);
+
+                    // Simply checking the unit quantities created
+                    Assert.True( simulatorDefinition.UnitQuantities.Count() == SeedData.SimulatorCreate.UnitQuantities.Count() );
```

Comment:
how about 

```
Assert.True(simulatorDefinition.UnitQuantities.SequenceEqual(unitQuantities));
                    Assert.True(simulatorDefinition.StepFields.SequenceEqual(SeedData.SimulatorCreate.StepFields));
                    Assert.True(simulatorDefinition.FileExtensionTypes.SequenceEqual(SeedData.SimulatorCreate.FileExtensionTypes));
                    Assert.True(simulatorDefinition.ModelTypes.SequenceEqual(SeedData.SimulatorCreate.ModelTypes));
                    
```

Reply:
does not work (always get false) .... i think thats because this needs implementations of `GetHashCode` & `Equals` on these types in the sdk.